forest/daemon/src/daemon.rs

```diff
@@ -464,13 +464,18 @@ async fn maybe_fetch_snapshot(
 ) -> anyhow::Result<Config> {
     if should_fetch_snapshot {
         let snapshot_path = default_snapshot_dir(&config);
+        let provider = SnapshotServer::try_get_default(&config.chain.name)?;
+        // FIXME: change this to `true` once zstd compressed snapshots is supported by
```

Comment:
Perhaps creating a tracking issue for this would be useful, so we don't forget about this.